src/handlers/mentions.rs

```diff
@@ -95,23 +109,30 @@ pub(super) async fn handle_input(
         IssueData::load(&mut client, &event.issue, MENTIONS_KEY).await?;
     // Build the message to post to the issue.
     let mut result = String::new();
-    for to_mention in &input.paths {
-        if state.data.paths.iter().any(|p| p == to_mention) {
-            // Avoid duplicate mentions.
-            continue;
-        }
-        let MentionsPathConfig { message, cc } = &config.paths[to_mention];
-        if !result.is_empty() {
-            result.push_str("\n\n");
-        }
-        match message {
-            Some(m) => result.push_str(m),
-            None => write!(result, "Some changes occurred in {to_mention}").unwrap(),
+    match input {
+        MentionsInput::HasBorsCommit => {
+            result = config.bors_commit_message.clone().unwrap_or_default();
```

Comment:
If the message isn't configured, we shouldn't post an empty comment/skip posting, but rather not have this logic at all.